packages/techdocs-common/src/stages/generate/types.ts

```diff
@@ -29,6 +29,7 @@ export type GeneratorConfig = {
   runIn: GeneratorRunInType;
   dockerImage?: string;
   pullImage?: boolean;
+  omitTechdocsPlugin?: boolean;
```

Comment:
Wonder if this could be a little less ambiguously named?  Like `omitTechdocsCoreMkdocsPlugin` or something?  Or, to build off of @andrewthauer's comment, nesting it, perhaps: `techdocs.generator.mkdocs.omitTechdocsCorePlugin` or something?

Reply:
Yup, that makes sense. Modifying.